website/src/components/views/wasap/WasapPage.tsx

```diff
@@ -87,6 +89,60 @@ export const WasapPageInner: FC<WasapPageProps> = ({ currentUrl }) => {
 
 export const WasapPage = withQueryProvider(WasapPageInner);
 
+const TotalCount = () => {
+    const { data, isPending, isError, error } = useQuery({
+        queryKey: ['aggregatedCount'],
+        queryFn: () => fetchTotalCount(wastewaterConfig.wasap.lapisBaseUrl),
+    });
+
+    return (
+        <div className='stat'>
+            <div className='stat-title'>Sequences</div>
+            <div className='stat-value text-base'>{isPending ? '…' : isError ? 'Error' : data.toLocaleString()}</div>
+            <div className='stat-desc text-wrap'>
+                {isPending
+                    ? 'Loading total sequences…'
+                    : isError
+                      ? error.message
+                      : 'The total number of sequenced samples'}
+            </div>
+        </div>
+    );
+};
+
+const formatDate = (d: string) =>
+    new Date(d).toLocaleDateString(undefined, { year: 'numeric', month: 'short', day: 'numeric' });
```

Comment:
I think we should fix the locale here so that it looks the same in everyone's browser. What do we want? US locale?
```suggestion
    new Date(d).toLocaleDateString('en-us', { year: 'numeric', month: 'short', day: 'numeric' });
```

Also usually we use this format: `2025-09-19`. The `month: 'short'` also looks nice but I wonder whether we should stick to our usual forma instead?

Reply:
... done. (see updated screenshot as well)